meetings.md

```diff
@@ -126,8 +126,8 @@ up-to-date [weekly availability schedules][schedules].
    [cultivator]: /roles.md#cultivator-aka-meeting-chair
    [bylaws-3]: /bylaws.md#3-member-meetings
    [bylaws-4]: /bylaws.md#4-board-of-directors
-   [scheduling]: /guides.md#scheduling-a-meeting
-   [hosting]: /guides.md#hosting-a-meeting
+   [scheduling]: /guides/meetings.md#scheduling-a-meeting
+   [hosting]: /guides/meetings.md##hosting-a-meeting
```

Comment:
This is supposed to have two ##, right?

Reply:
Good eye @writerly03 ! This is a typo. I've removed the extra #